pkg/reconciler/parallel/controller.go

```diff
@@ -61,5 +65,14 @@ func NewController(
 		Handler:    controller.HandleAll(impl.EnqueueControllerOf),
 	})
 
+	parallelGK := v1.SchemeGroupVersion.WithKind("Parallel").GroupKind()
```

Comment:
I think you can simplify this to:
```suggestion
	parallelGK := v1.Kind("Parallel")
```